packages/decent.js/docs/channel.md

```diff
@@ -0,0 +1,60 @@
+# const channel = client.channels.nth(0)
+Represents a single channel. Updated automatically.
+
+---
+
+Properties:
+
+* channel.id (string)
+* channel.name (string)
+* channel.deleted (boolean)
+
+The following properties are only provided if logged in:
+
+* channel.unreadMessageCount (?int)
+* channel.oldestUnreadMessageID (?string)
+
+---
+
+Extends [eventemitter3](https://npm.im/eventemitter3).
+
+## await channel.sendMessage(text: string) -> string messageID
+Sends a message to the channel, implicitly marking it as read.
```

Comment:
Could it be valuable to provide a boolean setting to enable/disable the implicit marking as read? Obviously defaulting to true would be best, but there's no reason not to give clients the option.

Reply:
>no reason

The _server_ marks it as read, not us.
It's not an API option.

Reply:
Should it be?

Reply:
@towerofnix

Reply:
Except for bots, what is the practical reason of *not* marking a channel as read once you send a message? @PullJosh